nigsp/operations/statistics.py

```diff
@@ -0,0 +1,195 @@
+import logging
+
+import numpy as np
+from mne.stats import permutation_t_test
+
+LGR = logging.getLogger(__name__)
+
+
+def ranktest(a, axis=None):
+    # Code adapted from `scipy`; ref: https://github.com/scipy/scipy/blob/v1.11.2/scipy/stats/_stats_py.py#L10123
```

Comment:
Rather than this kind of comment, this should go in a [`See Also`](https://numpydoc.readthedocs.io/en/latest/format.html#see-also) and [`Notes`](https://numpydoc.readthedocs.io/en/latest/format.html#notes) section.
In the `Notes` section, you should report the original copyright and licence and respect what that licence says (e.g. [here](https://github.com/physiopy/phys2bids/blob/76cc9ddcef0d450107249bf37a5758361a6703ad/phys2bids/utils.py#L272), although it shouldn't be in References). You should very explicitly say that this part of code respects the original BSD 3-Clause licence of Scipy and refer to it.